### PR TITLE
Add metadata to MultimodalContent

### PR DESCRIPTION
**Pull request by KaQuMiQ**

### CODE REVIEW

File: src/draive/types/audio.py [hunk 1]
```diff
@@ -11,12 +11,14 @@ class AudioURLContent(DataModel):
     mime_type: str | None = None
     audio_url: str
     audio_transcription: str | None = None
+    meta: dict[str, str | float | int | bool | None] | None = None
```
Comment (piotr-rarus) [inline src/draive/types/audio.py:14]: There's no possibility to pass a nested dict?

File: tests/test_model.py [hunk 2]
```diff
@@ -248,7 +248,12 @@ def test_basic_decoding() -> None:
     '"created": null, '
     '"content": {'
     '"parts": ['
-    '{"mime_type": null, "image_url": "https://miquido.com/image", "image_description": null}'
+    "{"
```
Comment (piotr-rarus) [inline tests/test_model.py:251]: Why all those test data are not fixtures?

Comment (piotr-rarus) [reply tests/test_model.py:251]: Right now those objects are module level. They can be mutated putting other tests at risk. Do we use random test order?

File: src/draive/types/audio.py (hunk 1 above)
Comment (KaQuMiQ) [reply src/draive/types/audio.py:14]: no and that is intentional to keep it concise - I don't want to put there a ton of information, the primary purpose of MultimodalContent is to make abstraction over multimodal content passed to or from LLMs, meta will be mostly ignored in that case, it is intended for additional processing and custom labeling of parts of the content i.e. to mark text section as a header and another one as a content

File: tests/test_model.py (hunk 2 above)
Comment (KaQuMiQ) [reply tests/test_model.py:251]: Those tests are yet to be refined and rewritten 😇

Comment (KaQuMiQ) [reply tests/test_model.py:251]: All of our data models are immutable - there is no risk of making accidental change there

**Pull request merged**